app/frontend/src/components/commons/Input/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { HTMLInputTypeAttribute } from 'react';
+
+import * as styles from './index.css';
+
+type InputProps = {
+  label: string;
+  maxLength: number;
+  placeholder?: string;
+  errorMessage?: string;
+  type?: HTMLInputTypeAttribute;
+  disabled?: boolean;
+  required?: boolean;
+};
+
+export function Input({
+  label,
+  type = 'input',
+  placeholder = '',
+  errorMessage = '',
+  disabled = false,
+  maxLength,
+  required = false,
+}: InputProps) {
+  return (
+    <div className={`${styles.container} ${errorMessage && styles.error} ${disabled && styles.disabled}`}>
+      <div className={styles.titleWrapper}>
+        <span className={styles.label}>
+          {label}
+          {required && <span className={styles.required}>*</span>}
+        </span>
+        <span className={styles.count}>0/{maxLength}</span>
+      </div>
+      <input
+        className={styles.input}
+        type={type}
+        placeholder={placeholder}
+        disabled={disabled}
+        maxLength={maxLength}
+        required
```

Comment:
p1 : required 속성이 빠진 것 같아요
```suggestion
        required={required}
```

Reply:
[71c70da](https://github.com/boostcampwm2023/web17_morak/pull/57/commits/71c70da6f2dbe2fe53ab5315590b8e721c4df523) 깃허브에서 바로 반영했는데 지원님과 함께 작성한 커밋으로 반영되네요 신기하군요